packages/ui/hooks/useDefaultValues.ts

```diff
@@ -51,48 +51,69 @@ export const useDefaultValues = <T extends StrapiModel>(
     if (!model || !fields) return {} as T
 
     const defaults = {} as any
-    const { date, createdAt, updatedAt, publishedAt } = model as Activity
 
-    const getDate = (date?: string | null, isDateTime?: true | 'amsterdam') => {
+    const convertDate = (
+      fieldName: string | number | symbol,
+      toDateOnly: boolean,
+      date?: string | null,
+    ) => {
       if (!date) return ''
 
       const dateTime = new Date(date)
```

Comment:
```suggestion
      const dateTime = parseISO(date)
```

Could you try parseISO if it fixes timeout problems without needing timezone difference calculation?

Otherwise you can merge it.